examples/capi-quickstart.yaml

```diff
@@ -1,128 +0,0 @@
-apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
is this file removed on purpose, and `aws-capi-quickstart.yaml` should be used instead?

Reply:
That's right. The `aws-capi-quickstart.yaml` has the quickstart manifest for aws.